src/row.js

```diff
@@ -62,6 +62,22 @@ export class Row {
     this.enemyFaction = row[47];
   }
 
+  getComposition = brackets => {
+    // We could have added all 5 classes and then filter out falsy values, but it would not be resilient to "ghost player" bugs
+    let arr = [this.enemyPlayerClass1, this.enemyPlayerClass2];
+    if (brackets.includes('5s'))
+      arr.push(
+        this.enemyPlayerClass3,
+        this.enemyPlayerClass4,
+        this.enemyPlayerClass5
+      );
+    else if (brackets.includes('3s')) arr.push(this.enemyPlayerClass3);
+    return arr
+      .filter(it => !!it)
```

Comment:
Use more meaninful variable names so we understand clearly what your are iterating on.